arelle/plugin/validate/ESEF_2022/Const.py

```diff
@@ -22,6 +22,7 @@
     "http://www.esma.europa.eu/taxonomy/[0-9-]{10}/esef_all.xsd"
     )
 
+esefCorNsPattern = re.compile(r"https?://www\.esma\.europa\.eu/taxonomy/[0-9-]{10}/esef_cor")
```

Comment:
Should this be pinned to a specific year?

Reply:
As a matter of fact, the latest taxonomy must be used (currently 2022), but I think it would be cumbersome to change every year that taxonomy...